types.go

```diff
@@ -93,12 +93,12 @@ type DeviceSubscriptionRequest struct {
 
 // SendNotificationRequest is representation of data to be sent to pushy service to create new notification
 type SendNotificationRequest struct {
-	To                  []string        `json:"to"`
-	Data                string          `json:"data"`
-	TimeToLive          int             `json:"time_to_live"`
-	IOSMutableContent   bool            `json:"mutable_content"`
-	IOSContentAvailable bool            `json:"content_available"`
-	IOSNotification     IOSNotification `json:"notification"`
+	To                  []string               `json:"to"`
+	Data                map[string]interface{} `json:"data"`
```

Comment:
There's a good chance that some people might not actually use JSON, that's the reason I didn't want to have map, if you have a map, you need to parse into text and send that to pushy. What if they want to send an XML, may I know the reason for this?

also build failed.

Reply:
`pushyMessage := pushy.SendNotificationRequest{
			To:   []string{<device_id>},
			Data: `{"message":"hello world"}`,
		}`
i have tried using it like this, this doesn't work with pushy API. i get this error - `Please supply the notification payload as a key-value object.`

Reply:
you can fix the build, I'll test it and I'll merge if it works,

Reply:
or let me check this weekend if there's some new change in pushy's API, I was using this for a while and then I stopped doing android.

Reply:
The TravisCI build is failing for this reason - `The command "go get -u golang.org/x/lint/golint" failed and exited with 2 during.`
I think this is the same reason for which the last commit on the master was failing.
Any idea on how can we fix this?